.circleci/config.yml

```
@@ -91,7 +91,37 @@ jobs:
       - persist_to_workspace:
           root: *working_directory
           paths:
-            - dist      
+            - dist
+            
+  check-gatsby-ssr:
+    <<: *defaults
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache: *restore_cache
+      - run: *npm_install
+      - save_cache: *save_cache
+
+      - run:
+          name: Change Permissions
+          command: sudo chown -R $(whoami) /usr/local
```

@dej611 do we really need this? This folder should be already accessible by user.

---

I tried without it [here](https://app.circleci.com/pipelines/github/italia/design-react-kit/161/workflows/85bae6d4-2365-40fc-84f7-41f145c3d5ae/jobs/2877) and it raises permission problems. Looking on Internet I found this solution. I'm open to alternatives that works :D

---

```suggestion
          name: Bind global package folder to local user
          command: |
              mkdir ~/.npm-packages
              npm config set prefix ~/.npm-packages
```
This should do the trick. We will use a local user folder to store all global packages. What do you think?